docs/administration-guide/08-configuration-settings.md

```diff
@@ -13,6 +13,9 @@ The **report timezone** sets the default time zone for displaying times. The tim
 
 *Setting the default timezone will not change the timezone of any data in your database*. If the underlying times in your database aren't assigned to a timezone, Metabase will use the report timezone as the default timezone.
 
+### Enable X-rays
+[X-rays](../users-guide/14-x-rays.md) are a great way to allow your users to quickly explore tables, question results, or interesting parts of charts. But if you'd prefer your users not use x-rays, you can turn them off here.
```

Comment:
Would it make sense to mention reasons why you want to turn them off here? (Data warehouse charges, etc) Might help people make an informed decision so they can decide whether or not to turn them off before they have a negative experience.